test/spec/video_spec.js

```diff
@@ -1,6 +1,7 @@
-import { isValidVideoBid } from 'src/video.js';
+import { isValidVideoBid, registerVastTrackers, insertVastTrackers, getVastTrackers, addImpUrlToTrackers } from 'src/video.js';
 import {hook} from '../../src/hook.js';
 import {stubAuctionIndex} from '../helpers/indexStub.js';
+import { MODULE_TYPE_ANALYTICS } from 'src/activities/modules.js';
```

Comment:
why are these imports needed if no changes are made in the tests ? 
The imports from `video.js` don't even exist
```suggestion
import { isValidVideoBid } from 'src/video.js';
import {hook} from '../../src/hook.js';
import {stubAuctionIndex} from '../helpers/indexStub.js';
```